examples/revisiting.py

```diff
@@ -156,30 +158,85 @@ def test(loader: DataLoader) -> float:
         return rmse
 
 
-if is_classification:
-    metric = 'Acc'
-    best_val_metric = 0
-    best_test_metric = 0
-else:
-    metric = 'RMSE'
-    best_val_metric = float('inf')
-    best_test_metric = float('inf')
-
-for epoch in range(1, args.epochs + 1):
-    train_loss = train(epoch)
-    train_metric = test(train_loader)
-    val_metric = test(val_loader)
-    test_metric = test(test_loader)
-
-    if is_classification and val_metric > best_val_metric:
-        best_val_metric = val_metric
-        best_test_metric = test_metric
-    elif not is_classification and val_metric < best_val_metric:
-        best_val_metric = val_metric
-        best_test_metric = test_metric
-
-    print(f'Train Loss: {train_loss:.4f}, Train {metric}: {train_metric:.4f}, '
-          f'Val {metric}: {val_metric:.4f}, Test {metric}: {test_metric:.4f}')
-
-print(f'Best Val {metric}: {best_val_metric:.4f}, '
-      f'Best Test {metric}: {best_test_metric:.4f}')
+if args.framework == "torch":
+    if is_classification:
+        metric = 'Acc'
+        best_val_metric = 0
+        best_test_metric = 0
+    else:
+        metric = 'RMSE'
+        best_val_metric = float('inf')
+        best_test_metric = float('inf')
+
+    for epoch in range(1, args.epochs + 1):
+        train_loss = train(epoch)
+        train_metric = test(train_loader)
+        val_metric = test(val_loader)
+        test_metric = test(test_loader)
+
+        if is_classification and val_metric > best_val_metric:
+            best_val_metric = val_metric
+            best_test_metric = test_metric
+        elif not is_classification and val_metric < best_val_metric:
+            best_val_metric = val_metric
+            best_test_metric = test_metric
+
+        print(
+            f'Train Loss: {train_loss:.4f}, '
+            f'Train {metric}: {train_metric:.4f}, '
+            f'Val {metric}: {val_metric:.4f}, Test {metric}: {test_metric:.4f}'
+        )
+
+    print(f'Best Val {metric}: {best_val_metric:.4f}, '
+          f'Best Test {metric}: {best_test_metric:.4f}')
+elif args.framework == "skorch-dataframe":
+    import numpy as np
```

Comment:
nit: I think we can move these imports to top as we will install these packages for pytorch frame